scripts/steps/perform_release_codegen.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -x
+
+# Code-generates the project, stages files in release position,
+# generates a new Package.swift
+# install_dependencies_on_ci.sh should be run before this script.
+
+# Clear the contents of the release folder so it can be rebuilt clean
+rm -rf release/*
+
+# Build the SDK components
+./gradlew -p codegen/sdk-codegen build
+
+# Move SDK components into place
+./gradlew -p codegen/sdk-codegen stageSdks
```

Comment:
thinking more into this, stageSdks should be the only task needed to tie things up. It must do the copying the SDK, merging models and generated the package.swift file.

This will make the developer feedback much faster, from the intellij, devs can build the codegen and stage. No other external commands are required.

Next commands would be just
swift build and swift test